.travis.yml

```diff
@@ -5,10 +5,17 @@ python:
 
 cache: pip
 
+env:
+  NODE_VERSION="6"
+
 install:
-  - pip install --upgrade pip setuptools
+  - nvm install $NODE_VERSION
+  - npm install --global ganache-cli
+  - wget https://github.com/ethereum/solidity/releases/download/v0.4.18/solc-static-linux && chmod +x ./solc-static-linux && sudo mv solc-static-linux /usr/bin/solc
+  - pip install --upgrade pip setuptools flake8
```

Comment:
should we also add `flake8` to setup?

Reply:
I think we should. I didn't want to be assumptive with the PR, but it doesn't really make sense for users to have to install it separately.